apps/explorer/src/utils/operator.ts

```diff
@@ -94,7 +94,7 @@ export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; perc
     totalAmount = new BigNumber(buyAmount)
   }
 
-  return { amount: executedAmount, percentage: new BigNumber(executedAmount.toFixed() / totalAmount.toFixed()) }
+  return { amount: executedAmount, percentage: executedAmount.div(totalAmount) }
```

Comment:
There where 2 issues.

This is 1, where the amounts were converted from BigNumber to string, divided, then turned into BigNumber.